pliers/extractors/text.py

```diff
@@ -13,14 +14,20 @@
 import pandas as pd
 import nltk
 from nltk.sentiment.vader import SentimentIntensityAnalyzer
-import sys
+import os
+import re
+from spacy.cli.download import download as spacy_download
```

Comment:
I think you can drop this line; `spacy` should already be in the namespace thanks to the `attempt_to_import` line below, which implicitly imports it (if it's installed)